client/app/tasks/schema/task.ts

```diff
@@ -7,11 +7,18 @@ const taskSchema = z.object({
     .trim()
     .min(1, { message: 'Title is required' })
     .max(100, { message: 'Title must be less than 100 characters' }),
-  date: z.date({ required_error: 'Date is required.' }),
-  assigneeId: z.number({
-    required_error: 'Assignee is required',
-  }),
-
+  date: z
+    .date()
+    .nullable()
+    .refine((date) => date !== null, {
+      message: 'Date is required.',
+    }),
```

Comment:
I got an error if you try to update without changing the date
<img width="600" alt="Screenshot 2024-03-21 at 3 09 34 PM" src="https://github.com/HAK2024/fairy-share/assets/66394413/e54e5186-c733-4f44-a53a-da661675dd17">

Reply:
Fixed it!
https://github.com/HAK2024/fairy-share/pull/84/commits/b48a68a2011241d21f7df287e9b503538f38ba81